Create Large Test Files.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Creating Large HMDA Submission Test Files
+# 
+# This notebook demonstrates a function for creating large HMDA submission test files using 1000 row clean files for bank 1 and bank 0. The function concatenates the files together, creates unique Universal Loan Identifiers (ULIs) with a check digit, and generates a submission file with a specified number of LAR rows. 
+
+# In[280]:
+
+
+#The following code imports the required packages.
+
+import json
+import os
+import pandas as pd
+import random
+import string
+import time
+import yaml
+import numpy as np
+
+
+# In[281]:
+
+
+#The function below generates a random character string with a given length to create loan identifiers.
+def char_string_gen(length):
```

Comment:
Are these two functions modified from the code in https://github.com/cfpb/hmda-test-files/blob/master/python/lar_generator.py? If not, is it possible to import them instead?